src/manipulation.js

```diff
@@ -159,6 +159,10 @@ function fixInput( src, dest ) {
 }
 
 jQuery.extend({
+	htmlPrefilter: function( html ) {
```

Comment:
So user would need to redefine this method? And if their would, they would loose, `html.replace( rxhtmlTag, "<$1></$2>" )` User would need to duplicate it? Isn't that null out the whole point of that function?